components/Dashboard/Post/PostAuthorCard.tsx

```diff
@@ -39,6 +66,15 @@ const PostAuthorCard: React.FC<PostAuthorCardProps> = ({ author }) => {
           </a>
         </Link>
         <p className="author-name">{author.name || author.handle}</p>
+        {currentUser && currentUser.id !== author.id && (
+          <Button
+            variant={ButtonVariant.Link}
+            onClick={hasFollowedAuthor ? handleUnfollowUser : handleFollowUser}
+            loading={followLoading || unfollowLoading}
+          >
+            {hasFollowedAuthor ? 'Unfollow' : 'Follow'}
```

Comment:
Little thing but ideally these would be translated

Reply:
Great catch! Done!